src/cloud/social/provider.ts

```diff
@@ -58,6 +58,14 @@ interface SavedContact {
   description?: string;
 }
 
+var UserStatus = {
```

Comment:
This user.status field is kinda annoying.  I entered https://github.com/uProxy/uproxy/issues/2158 to clean this up.  I think the slightly longer term solution is to convert user.status to a string field, and then emit string values like "FRIEND" - this is the same as client.status and the way FreedomJS expects things to work (a little annoying in TypeScript though because TypeScript enums usually evaluate to numbers... actually they are maps like ```{"FRIEND": 0, 0: "FRIEND"}```).

I don't have a strong feeling about moving cloud to the uProxy repo, but I'm not sure it's worth doing just for this enum.